File: 3396-valid-word/valid-word.cpp
class Solution {
public:
    bool isValid(string word) {
        if(word.length()<3) return false;
        int vow=0,cons=0;
        for(char it:word){
            if(it=='a'||it=='e'||it=='i'||it=='o'||it=='u'||it=='A'||it=='E'||it=='I'||it=='O'||it=='U')
                vow++;
            else if(isalpha(it))
                cons++;
            else if(it>='0' && it<='9')
                continue;
            else
                return false;
        }
        if(vow>0 && cons>0)
            return true;
        return false;
    }
};

File: 3396-valid-word/valid-word.py
class Solution:
    def isValid(self, word: str) -> bool:
        if len(word) < 3:
            return False
        vow=0
        cons=0
        for it in word :
            if it in 'aeiouAEIOU':
                vow+=1
            elif it.isalpha():
                cons+=1
            elif(it>='0' and it<='9'):
                continue
            else:
                return False
        if(vow>0 and cons>0):
            return True
        return False